udata/tests/apiv2/test_datasets.py

```diff
@@ -199,22 +199,30 @@ def test_get_dataset_extras(self):
         assert data['test::extra'] == 'test-value'
 
     def test_update_dataset_extras(self):
-        self.dataset.extras = {'test::extra': 'test-value', 'test::extra-second': 'test-value-second'}
+        self.dataset.extras = {
+            'test::extra': 'test-value',
+            'test::extra-second': 'test-value-second',
+        }
         self.dataset.save()
 
         data = ['test::extra-second', 'another::key']
         response = self.put(url_for('apiv2.dataset_extras', dataset=self.dataset), data)
         self.assert400(response)
         assert response.json['message'] == 'Wrong payload format, dict expected'
 
-        data = {'test::extra-second': 'test-value-changed', 'another::key': 'another-value'}
+        data = {
+            'test::extra-second': 'test-value-changed',
+            'another::key': 'another-value',
+            'test::none': None,
```

Comment:
I don't think we test that it unsets a key here, right?

Reply:
True! I added a test.